### ISSUE TITLE
Opensourcepos is not working on localhost

### ISSUE THREAD

**Issue by muhammadubaid001** [opensourcepos/opensourcepos#2881]
Hi there, I have downloaded the latest version of opensourcepos and it is not working on my localhost showing me the following.

![opensourcepos](https://user-images.githubusercontent.com/39967022/86532185-4e5f2a00-bee1-11ea-94ae-7bfe7e4efd19.png)

**Comment by odiea** [comment]
And do you have the correct database settings also read the wiki for more suggestions

**Comment by muhammadubaid001** [comment]
Yes, it was working till today but now it is showing me the above HTTP error.

**Comment by SteveIreland** [comment]
I see you are running WampServer.  Did you check the WampServer logs?  Are they indicating anything that might be useful in debugging the problem?

**Comment by abid1208** [comment]
You might be using the wrong code base. It happened to me and when I used the code from the release version, the error resolved

**Comment by mwillis72** [comment]
Am facing the same after installation

**Comment by asakpke** [comment]
> Hi there, I have downloaded the latest version of opensourcepos and it is not working on my localhost showing me the following...

Open `index.php` file at `public` folder and modify this line `define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'production');` to `define('ENVIRONMENT', 'development');`. Then reload page & notice any error, warning etc and fix them.

**Comment by jekkos** [comment]
@muhammadubaid001 did you manage to fix?
